css/styles.css

```diff
@@ -41,11 +41,13 @@
 /* @group Columns */
 
 .ab-wrap {
-    margin: 20px 0 0 -30px;
+    margin: 0 0 0 -10px;
+    padding: 20px 0 0 0;
```

Comment:
You might actually have a point here. But since I was not fully sure of possible consequences these changes might have, so I reverted them back to you commit 2d193ccc36be60025bd5d8d82f6238b0203b5ffb